openslides/locale/tinymce/cs.js

```diff
@@ -0,0 +1,219 @@
+tinymce.addI18n('cs',{
```

Comment:
Hm. That looks ugly. What about creating an own repository so that we do not have to keep these LGPL-language files in our (MIT-licensed) core.